modules/standard/Math.chpl

```diff
@@ -615,6 +615,9 @@ module Math {
 
   /* Computes the mod operator on the two arguments, defined as
      ``mod(x,y) = x - y * floor(x / y)``.
+
+     The result is always >= 0 if `n` > 0.
+     It is an error if `n` == 0.
```

Comment:
It seems a little unfortunate that the mod demonstration in the line above uses different names for the arguments than the comment you've added.  Would you mind unifying so that the line above matches the argument names in the declaration?